torch/distributed/elastic/rendezvous/filestore_rendezvous_backend.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import logging
+from typing import Tuple, Optional
+from torch.distributed import Store, FileStore
+from .dynamic_rendezvous import RendezvousBackend, Token
+from .api import RendezvousParameters
+
+
+log = logging.getLogger(__name__)
+
+
+class FileStoreRendezvousBackend(RendezvousBackend):
```

Comment:
In fact you do not need a `FileStoreRendezvousBackend`. The `C10dRendezvousBackend` works with any store type as long as it supports the `compare_set()` function. `FileStore` recently introduced support for `compare_set`, so all you have to do is change [this line](https://github.com/pytorch/pytorch/blob/master/torch/distributed/elastic/rendezvous/c10d_rendezvous_backend.py#L201).